Directory.Build.targets

```diff
@@ -7,6 +7,21 @@
     <ImportDirectoryBuildTargets>false</ImportDirectoryBuildTargets>
   </PropertyGroup>
 
+  <!--
+  When .NET gets built from source, make the SDK aware there are bootstrap packages
+  for Microsoft.NETCore.App.Runtime.<rid> and Microsoft.NETCore.App.Crossgen2.<rid>.
+  -->
+  <ItemGroup Condition=" '$(DotNetBuildFromSource)' == 'true' ">
```

Comment:
Does this logic need to evaluate before the Arcade and Microsoft.Net Sdk's targets files are evaluated? If not, please move this after the imports as we usually not put anything before the Sdk import.